geopandas/tools/tests/test_sjoin.py

```diff
@@ -136,7 +136,7 @@ def test_deprecated_op_param_nondefault_predicate(self, dfs, op, predicate):
         if op != predicate:
             warntype = UserWarning
             match = (
-                "`predicate` will be overridden by the value of `op`"
+                "`predicate` will be overridden by the value of `op`"  # noqa: ISC003
```

Comment:
```suggestion
                + "`predicate` will be overridden by the value of `op`"
```
Given the lines below use plus, may as well be consistent?

Reply:
What do you mean? I can't start the first line of multiline string with a plus operator.

Reply:
ah sorry, I clearly misread what was going on. The warning itself is suggesting that we should remove all the pluses I think but happy with just ignoring.